sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -359,6 +420,15 @@ def training_client():
         yield client
 
 
+@pytest.fixture
```

Comment:
Do you need separate fixture just for wait_for_job_condition or we can re-use [Training client like for other tests](https://github.com/kubeflow/training-operator/blob/db41d3f05090387e9bf7a57375bbc20303bcf5b2/sdk/python/kubeflow/training/api/training_client_test.py#L404) ?

Reply:
I separate fixture for `wait_for_job_condition` because it needs to mock function `get_job`. I'm not sure whether it will affect other UTs but I can have a try:)